.gitlint

```diff
@@ -19,6 +19,10 @@ max-line-count=200
 [title-max-length]
 line-length=72
 
+[title-starts-with-subsystem]
+regex = ^(([^:]+):)+ (.+)$
```

Comment:
Perhaps there's a couple of extra parens here. Also, we probably don't want `foo:bar:baz: Do good`, but rather `foo: bar: baz: Do better` (at least the latter how we do it so far).